src/app/contextBroker/contextBroker.cpp

```diff
@@ -1485,8 +1513,6 @@ int main(int argC, char* argV[])
     LM_X(1, ("dbName too long (max %d characters)", DB_NAME_MAX_LEN));
   }
 
-  LM_I(("Orion Context Broker is running"));
```

Comment:
This removal is part of the cache merge from 0.24.0 sub-branch? Or a modification on the code after merging?